app/tests/integration/nimbus/test_desktop_client_integrations.py

```diff
@@ -190,3 +199,108 @@ def test_check_telemetry_enrollment_unenrollment(
     else:
         if control is not False:
             assert False, "Experiment unenrollment was never seen in Ping Data"
+
+
+@pytest.mark.nimbus_integration
+@pytest.mark.xdist_group(name="group2")
+def test_check_telemetry_enrollment_unenrollment_pref_flip(
+    base_url,
+    selenium,
+    kinto_client,
+    slugify,
+    experiment_name,
+    create_desktop_experiment,
+    experiment_default_data,
+    check_ping_for_experiment,
+    telemetry_event_check
+):
+    _row_locator = (By.CSS_SELECTOR, "tr > td > span > span")
+    _search_bar_locator = (By.ID, "about-config-search")
+
+    requests.delete("http://ping-server:5000/pings")
+    targeting = helpers.load_targeting_configs()[0]
+    experiment_slug = str(slugify(experiment_name))
+    experiment_default_data["targetingConfigSlug"] = targeting
+    experiment_default_data["featureConfigId"] = 9
+    experiment_default_data["referenceBranch"] = {
+        "description": "reference branch",
+        "name": "Branch 1",
+        "ratio": 99,
+        "featureEnabled": True,
+        "featureValue": '{"value": "test_string_automation"}',
+    }
+    experiment_default_data["treatmentBranches"] = [
+        {
+            "description": "treatment branch",
+            "name": "Branch 2",
+            "ratio": 1,
+            "featureEnabled": False,
+            "featureValue": "",
+        }
+    ]
+    create_desktop_experiment(
+        experiment_slug,
+        "desktop",
+        targeting,
+        experiment_default_data,
+    )
+    wait = WebDriverWait(selenium, 10)
+    selenium.get("about:config")
+    search_bar = wait.until(EC.presence_of_element_located(_search_bar_locator))
+    search_bar.send_keys("nimbus.qa.pref-1")
+    wait.until(EC.presence_of_element_located(_row_locator))
+    elements = selenium.find_elements(*_row_locator)
+    assert "default" in [element.text for element in elements]
```

Comment:
You didn't want to pull this out into a page class?

Reply:
mmm, nahh, since we are just using it for this test, and this test only, I think this is fine. It isn't pretty but it's just for this test.

Reply:
Yeah, it just helps isolate concerns and makes the test nice and succinct.  Just a suggestion.